app/services/subscriptions/terminate_service.rb

```diff
@@ -111,17 +111,31 @@ def bill_subscription
           subscription.terminated_at,
           invoicing_reason: :subscription_terminating
         )
-        BillNonInvoiceableFeesJob.set(wait: 2.seconds).perform_later([subscription], subscription.terminated_at)
+        BillNonInvoiceableFeesJob.set(wait: 2.seconds).perform_later(
```

Comment:
should this wait not just be a `after_commit`? 🤔

Reply:
That is a good question... queuing these jobs is not wrapped within a DB transaction, the same for another job within this class... maybe a left over from a older implementation or just a fix for a race condition sidekiq vs Postgres.

Reply:
okay, but either it's committed and we don't need the wait or it's not and we just need to use `after_commit` no?